compiler+runtime/test/cpp/jank/read/lex.cpp

```diff
@@ -1336,5 +1305,100 @@ namespace jank::read::lex
         }
       }
     }
+    
+    TEST_CASE("UTF-8")
+    {
+      SUBCASE("UTF-8 symbol")
```

Comment:
You're already in the UTF-8 test case, so no need to repeat UTF-8 in each sub case name.